tests/Paket.Tests/TestHelpers.fs

```diff
@@ -9,11 +9,34 @@ open System.Xml
 open System.IO
 open Paket.Domain
 
-let PackageDetailsFromGraph (graph : seq<string * string * (string * VersionRequirement) list>) sources groupName (package:PackageName) (version:SemVerInfo) = 
+type GraphDependency = string * VersionRequirement * FrameworkRestrictions
+
+type DependencyGraph = list<string * string * (GraphDependency) list>
```

Comment:
Most of the diff is because I changed this (test-only) graph structure to support dependency groups...